lib/travis/logs/services/aggregate_logs.rb

```diff
@@ -118,14 +119,13 @@ def aggregate_log(log_id)
         end
 
         private def log_empty?(log_id)
-          log = database.log_for_id(log_id)
-          if log[:content].nil? || log[:content].empty?
-            Travis.logger.warn(
-              'aggregating',
-              action: 'aggregate', log_id: log_id, result: 'empty'
-            )
-            true
-          end
+          content = database.log_for_id(log_id)[:content]
```

Comment:
This would raise a `NoMethodError` if the log couldn't be found, I think? Looks like the old code did that too, though.